Flaky test: `chainerx_tests/unit_tests/routines_tests/test_creation.py::test_TrilTriu`

https://travis-ci.org/chainer/chainer/jobs/563001476

>`FAIL tests/chainerx_tests/unit_tests/routines_tests/test_creation.py::test_TrilTriu_param_4_{k=1, shape=(4, 3)}[float16-native:0]`

> https://travis-ci.org/chainer/chainer/jobs/563001476
>
> >`FAIL tests/chainerx_tests/unit_tests/routines_tests/test_creation.py::test_TrilTriu_param_4_{k=1, shape=(4, 3)}[float16-native:0]`


```
chainer.testing.function_link.FunctionTestError: backward is not implemented correctly

(caused by)
AssertionError: check_backward failed (eps=0.001 atol=1e-05 rtol=0.005)
inputs[0]:
array([[0.44628906, 0.87988281, 0.80273438],
       [0.76269531, 0.515625  , 0.31884766],
       [0.515625  , 0.71240234, 0.45703125],
       [0.98535156, 0.85986328, 0.31469727]], shape=(4, 3), dtype=float16, device='native:0')
grad_outputs[0]:
array([[0.46289062, -0.21643066, 0.6953125 ],
       [-0.00450897, 0.24365234, -0.01101685],
       [-0.06964111, -0.31298828, 0.03344727],
       [0.71582031, 0.52880859, -0.34594727]], shape=(4, 3), dtype=float16, device='native:0')
grad_outputs[1]:
array([[0.46923828, -0.34301758, -0.67724609],
       [0.14941406, -0.15942383, -0.8774414 ],
       [0.5727539 , 0.63085938, 0.56005859],
       [-0.1763916 , 0.94189453, -0.60644531]], shape=(4, 3), dtype=float16, device='native:0')
directions[0]:
array([[0.25658088, 0.36267571, -0.38868463],
       [0.66351429, 0.16558299, 0.1762533 ],
       [-0.30980999, -0.16816176, -0.09785577],
       [-0.11582871, -0.07045690, 0.02461404]], shape=(4, 3), dtype=float64, device='native:0')
gradients (numeric):  array(0.00211420, shape=(), dtype=float64, device='native:0')
gradients (backward): array(0.00203227, shape=(), dtype=float64, device='native:0')

x: numeric gradient, y: backward gradient
Not equal to tolerance rtol=0.005, atol=1e-05

(mismatch 100.0%)
 x: array(0.002114)
 y: array(0.002032)

assert_allclose failed: 
  shape: () ()
  dtype: float64 float64
  i: (0,)
  x[i]: 0.0021141975921907946
  y[i]: 0.0020322738864613776
  relative error[i]: 0.04031135088394194
  absolute error[i]: 8.192370572941698e-05
x: 0.0021142
y: 0.00203227
```

--- a/tests/chainerx_tests/unit_tests/routines_tests/test_creation.py
+++ b/tests/chainerx_tests/unit_tests/routines_tests/test_creation.py
@@ -1252,8 +1252,9 @@ class TestTrilTriu(op_utils.NumpyOpTest):
         self.dtype = float_dtype
         # backward with float16 sometimes does not pass tests with default rtol
         if self.dtype == 'float16':
-            self.check_backward_options.update({'rtol': 5e-3})
-            self.check_double_backward_options.update({'rtol': 5e-3})
+            self.check_backward_options.update({'rtol': 2e-3, 'atol': 2e-3})
+            self.check_double_backward_options.update(
+                {'rtol': 2e-3, 'atol': 2e-3})
 
     def generate_inputs(self):
         a = numpy.random.random(self.shape).astype(self.dtype)

